### DISCUSSION TITLE
Beatmap search filters should be improved similar to Discord

### DISCUSSION THREAD

**Discussion by finesseandstyle** [ppy/osu#28059] [Ideas] [answered]
Let's say I want to search "me & u". This is the result I get:

![image](https://github.com/ppy/osu/assets/41844929/54910749-73ca-43fc-8ffe-ab46fd47e074)

Now, I would like to not scroll through 300 maps (or potentially thousands of them depending on the term) to find the one I want, search needs some serious improving when looking for just the name or anything else, cause sometimes the thing you're searching for is very generic and appears on every map.

Yes I know you can put in a filter like so title="me & u" but I had to look that up, the point is, I shouldn't, the game should at the very least put a help popup to show you all the filters next to the feature you're using.

osu could implement discord's very handy method of filters, where it contextually appears, no wiki or searching required.
 
![image](https://github.com/ppy/osu/assets/41844929/6d1e556f-726a-4a96-b45f-f31d07672a84)

Another suggestion is selecting a beatmap's title or other parameter to automatically use the search with that name

![image](https://github.com/ppy/osu/assets/41844929/b52e5db6-9973-4bde-a0fc-3fb8df373422)

So if I select Title it would replace the search with title="me & u"

**Comment by Joehuu** [comment, marked as answer]
When the [new song select wedge is implemented](https://github.com/ppy/osu/issues/22204), it'll be possible to click the title/artist on the top left to search for similar ones.

**Reply by finesseandstyle** [reply]
Awesome, I was trying to search for song select/search topics but couldn't find it
